src/test/java/org/kie/processmigration/test/persistence/scripts/util/SQLScriptUtil.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.processmigration.test.persistence.scripts.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.kie.processmigration.test.persistence.scripts.DatabaseType;
+
+import static org.kie.processmigration.test.persistence.scripts.DatabaseType.POSTGRESQL;
+import static org.kie.processmigration.test.persistence.scripts.DatabaseType.SQLSERVER;
+
+/**
+ * Contains utility methods for working with SQL script.
+ */
+public final class SQLScriptUtil {
+
+    private static final String REGEX_OR = "|";
+    /**
+     * Standard SQL command delimiter.
+     */
+    private static final String DELIMITER_STANDARD = ";";
+    /**
+     * Delimiter used in MS SQL Server database systems.
+     */
+    private static final String DELIMITER_MSSQL = "GO";
+    /**
+     * Delimiter used in PostGreSQL database systems.
+     */
+    private static final String DOLLAR_QUOTED_BLOCK = "\\$.*\\$";
+    /**
+     * Delimiter used in DB2 BPMS 6.0-to-6.1 upgrade scripts.
+     */
+    private static final String DELIMITER_ALTER = "@";
```

Comment:
Do we need all of these delimiters here? I don't think some of those are used.

Reply:
Done